### ISSUE TITLE
Is this still being worked on?

### ISSUE THREAD

**Issue by mtboss124** [NotImplementedLife/FlipnoteDesktop#3]
i was wondering if this was still being worked on just that

**Comment by NotImplementedLife** [comment]
Yes, the project is still active. I'm trying to solve some annoying performance issues, that's why it's taking so long.

**Comment by BoxBoiWasTaken2** [comment]
This project is gonna be awesome, I've been hoping for someone to make a decent flipnote creator for PC.

**Comment by Dara03525** [comment]
hello, is this programme useable? Im new to this thing and github. I just found this flipnote net googling flipnote alternatives. If there is an app or website where i can use this, please let me know, as im so lose.

**Comment by NotImplementedLife** [comment]
> hello, is this programme useable? Im new to this thing and github. I just found this flipnote net googling flipnote alternatives. If there is an app or website where i can use this, please let me know, as im so lose.

Hi Dara! And welcome to github! 

I've just uploaded a prerelease version containing the current state of the project for you (and others) to try and test things out.

Go to this link and download the zip file from the Assets section: https://github.com/NotImplementedLife/Flipnote.NET/releases/tag/pre_240917. Then, unzip the archive and run "FlipnoteDotNet.exe". You'll need a Windows system with .NET 6.0 in order to run the application.

Please remember that this is still a work in progress and the number of things you can do in the app is quite limited for the moment. I would greatly appreciate your feedback if you try Flipnote.NET. Also, if you have problems with launching the app or encounter any bugs while using it, please let me know!
